scripts/dts/gen_defines.py

```diff
@@ -516,10 +516,14 @@ def write_vanilla_props(node):
             # DT_N_<node-id>_P_<prop-id>
             macro2val[macro] = val
 
+        spec = prop.spec
+        if spec.string_tokenizable:
```

Comment:
I think you can just use `if prop.spec.type == 'string'`; am I missing something or do you really need a new attribute like this? If not, let's keep it simple and compare the type directly here.